web/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/web/rest/person/PersonController.kt

```diff
@@ -125,7 +126,7 @@ class PersonController @Autowired constructor(private val kjerneinfoService: Per
                             "motpartsRolle" to it.motpartsRolle,
                             "motpartsPersonident" to it.motpartsPersonident,
                             "motpartsPersonNavn" to navn,
-                            "omraade" to it.omraader,
+                            "omraade" to it.omraader.map { StandardKodeverk::getArkivtemaNavn },
```

Comment:
Dette blir nok ikke helt riktig.
`getArkivtemaNavn` er ikke en statisk metode, så metode-referansen må forholde seg til en instans av `StandardKodeverk`.

Isåfall burde følgende være gyldig: `"omraade" to it.omraader.map(standardKodeverk::getArkivtemaNavn)`